setup.py

```diff
@@ -70,5 +70,5 @@ def stream_requirements(fd):
         'Programming Language :: Python :: 3.5',
     ],
     test_suite='tests',
-    tests_require=REQUIREMENTS_TEST,
+    tests_require=REQUIREMENTS_TEST
```

Comment:
it was (trivially) better before :man_shrugging: 

```suggestion
    tests_require=REQUIREMENTS_TEST,
```